### PR TITLE
convert a bunch of get/set to plain fields

### PR DESCRIPTION
**Pull request by glabute**
### Purpose of this PR

CMCL-330 - performance optimization

### Testing status

- [ ] Added an automated test
- [ ] Passed all automated tests
- [x] Manually tested 

### Documentation status

- [ ] Updated [CHANGELOG](https://keepachangelog.com/en/1.0.0/)
- [ ] Updated README (if applicable)
- [ ] Commented all public classes, properties, and methods
- [ ] Updated user documentation

### Samples status

- [ ] Re-packed Extras~/CinemachineExamples.unitypackage

### Technical risk

low

### Comments to reviewers

It's a benign API change only.  It should have no effect on client code.

### CODE REVIEW

**Review by mjmvisser — APPROVED** [review]
looks good to me

File: Runtime/Behaviours/CinemachineBrain.cs [hunk 1]
```diff
@@ -643,8 +648,8 @@ private void ProcessActiveCamera(float deltaTime)
                     activeCamera.UpdateCameraState(DefaultWorldUp, deltaTime);
                 }
                 // Apply the vcam state to the Unity camera
-                PushStateToUnityCamera(
-                    SoloCamera != null ? SoloCamera.State : mCurrentLiveCameras.State);
```
[the remark this answers is not in the document]
Comment (glabute) [reply Runtime/Behaviours/CinemachineBrain.cs:647]: yes

File: Runtime/Behaviours/CinemachineBrain.cs [hunk 2]
```diff
@@ -606,7 +606,12 @@ public void ReleaseCameraOverride(int overrideId)
         private void ProcessActiveCamera(float deltaTime)
         {
             var activeCamera = ActiveVirtualCamera;
-            if (activeCamera == null)
+            if (SoloCamera != null)
+            {
+                var state = SoloCamera.State;
+                PushStateToUnityCamera(ref state);
+            }
+            else if (activeCamera == null)
```
[the remark this answers is not in the document]
Comment (glabute) [reply Runtime/Behaviours/CinemachineBrain.cs:614]: yes

**Review by gaborkb — APPROVED** [review]
Created a PR this branch with a few small extra cleanups.

**Pull request merged**